tools/cli/adaparse.cpp

```diff
@@ -112,17 +114,11 @@ int piped_file(Callable&& adaparse_print, const cxxopts::ParseResult result,
 
       ada::result<ada::url_aggregator> url = ada::parse(line);
       if (!url) {
-        adaparse_print("Invalid URL:{}\n", line);
-
-      } else if (result.count("get")) {
-        std::string get_part = result["get"].as<std::string>();
+        fmt::print(stderr, "Error opening file: {}\n", strerror(err)); shit
```

Comment:
It looks like I'm supposed to catch this, but I am not sure what is the intended improvement over the original? Was it to print whatever was causing a formatted URL?  

 The former seems to match what the new documentation is intending.  I tried playing around with it a bit, but I'd probably have to take a deeper look as to how url keeps track of errors.

Reply:
Ah. AH. My bad.

Reply:
See https://github.com/Nick-Nuon/ada/pull/6

I code with copilot and it will overwrite code if I am not careful.

Reply:
haha, yeah I suspected it had something to do with AI.   :p